recipes/sphinx_gmt/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "sphinx_gmt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99889d655237928bf6c3d4d1b7abef2e8252342a0cf203f2f88eccef2db13a62
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - python
```

Add your python constraint here

```suggestion
    - python >=3.5
```